etcdctl/READMEv3.md

```diff
@@ -370,6 +370,64 @@ Simple reply
 [mirror]: ./doc/mirror_maker.md
 
 
+### SNAPSHOT [subcommand]
+
+SNAPSHOT takes a point-in-time snapshot of etcd and restores it into a fresh cluster.
```

Comment:
it seems like snapshot and restore happens atomically. Probably we shall rephrase? SNAPSHOT provides the functionality to take a point-in-time snapshot of a running etcd server and to restore from the saved snapshot into a fresh cluster.